cypress/test/newsroom/FeaturedBlogItem.cy.js

```diff
@@ -0,0 +1,272 @@
+            "lvl": 3,
+            "i": 16,
+            "seen": 0
+        },
+        {
+            "content": "What's next with Twitter",
+            "slug": "whats-next-with-twitter",
+            "lvl": 3,
+            "i": 17,
+            "seen": 0
+        },
+        {
+            "content": "GitHub",
+            "slug": "github",
+            "lvl": 2,
+            "i": 18,
+            "seen": 0
+        },
+        {
+            "content": "Most popular repositories",
+            "slug": "most-popular-repositories",
+            "lvl": 3,
+            "i": 19,
+            "seen": 0
+        },
+        {
+            "content": "Traffic growth",
+            "slug": "traffic-growth",
+            "lvl": 3,
+            "i": 20,
+            "seen": 0
+        },
+        {
+            "content": "What's next with GitHub",
+            "slug": "whats-next-with-github",
+            "lvl": 3,
+            "i": 21,
+            "seen": 0
+        },
+        {
+            "content": "npm",
+            "slug": "npm",
+            "lvl": 2,
+            "i": 22,
+            "seen": 0
+        },
+        {
+            "content": "Top packages",
+            "slug": "top-packages",
+            "lvl": 3,
+            "i": 23,
+            "seen": 0
+        },
+        {
+            "content": "What's next with npm",
+            "slug": "whats-next-with-npm",
+            "lvl": 3,
+            "i": 24,
+            "seen": 0
+        },
+        {
+            "content": "YouTube",
+            "slug": "youtube",
+            "lvl": 2,
+            "i": 25,
+            "seen": 0
+        },
+        {
+            "content": "Watch time goes crazy",
+            "slug": "watch-time-goes-crazy",
+            "lvl": 3,
+            "i": 26,
+            "seen": 0
+        },
+        {
+            "content": "Most popular videos",
+            "slug": "most-popular-videos",
+            "lvl": 3,
+            "i": 27,
+            "seen": 0
+        },
+        {
+            "content": "What's next with YouTube",
+            "slug": "whats-next-with-youtube",
+            "lvl": 3,
+            "i": 28,
+            "seen": 0
+        },
+        {
+            "content": "Open Collective",
+            "slug": "open-collective",
+            "lvl": 2,
+            "i": 29,
+            "seen": 0
+        },
+        {
+            "content": "Ultimate questions - how many people use the spec?",
+            "slug": "ultimate-questions---how-many-people-use-the-spec",
+            "lvl": 2,
+            "i": 30,
+            "seen": 0
+        },
+        {
+            "content": "Final words",
+            "slug": "final-words",
+            "lvl": 2,
+            "i": 31,
+            "seen": 0
+        }
+    ],
+    "readingTime": 18,
+    "sectionSlug": "/blog",
+    "sectionWeight": 0,
+    "id": "pages/blog/2020-summary.md",
+    "isIndex": false,
+    "slug": "/blog/2020-summary"
+}
+
```

Comment:
Store this object as part of different static file where you can use this content to test BlogLayout and DocLayout as well.

Reply:
done

Reply:
Once this PR is merged , will add for docsLayout and BlogLayout as well , else it will cause a lot of conflicts .